templates/codelists/version.html

```diff
@@ -12,10 +12,16 @@ <h3>
   {% if clv.is_draft %}
   <span class="badge badge-primary">Draft</span>
   {% endif %}
-
 </h3>
 <br />
 
+{% if codelist.coding_system_id == "snomedct" %}
+<div class="alert alert-primary mb-5" role="alert">
+  SNOMED CT codelists have been automatically generated. For more information
```

Comment:
"SNOMED CT codelists have been automatically generated and have not been reviewed for accuracy."

Reply:
Fixed in a50db7d